soroban-env-host/src/test/token.rs

```diff
@@ -2208,7 +2339,7 @@ fn test_asset_token_classic_balance_boundaries_with_limit_and_liabilities() {
 #[test]
 fn test_asset_token_classic_balance_boundaries_large_values() {
     test_wrapped_asset_classic_balance_boundaries(
-        i64::MAX,
+        i64::MAX - i64::MAX / 5,
```

Comment:
Why did this change?

Reply:
This test specifies `i64::MAX / 5` amount of buying liabilities, which means the max balance should be `i64::MAX - i64::MAX / 5`.

Reply:
I added a step in `test_wrapped_asset_classic_balance_boundaries` that sends the balance back and it fails there.